src/app.js

```diff
@@ -1 +1,21 @@
 'use strict';
+
+const path = require('path');
+const { copyFile } = require('./copyFile');
+
+const cp = () => {
+  const [source, dest] = process.argv.slice(2);
+
+  if (!source || !dest) {
+    throw new Error(
+      'You have to pass source and dest files path'
+    );
+  }
+
+  const sourcePath = path.join(__dirname, source);
```

Comment:
The source file path is being joined with __dirname. This might cause issues if an absolute path is passed. Consider checking if the provided path is relative before joining it with __dirname.